src/libs/setShouldShowMainComposeInputKeyboardAware/index.native.js

```diff
@@ -8,11 +8,17 @@ export default (shouldShow) => {
         keyboardDidHideListener = null;
     }
 
-    if (!shouldShow || !Keyboard.isVisible()) {
+    if (!shouldShow) {
         Composer.setShouldShowComposeInput(shouldShow);
```

Comment:
```suggestion
        Composer.setShouldShowComposeInput(false);
```

Reply:
Split logic and added comment

Reply:
This is not about split logic. Please check again @s-alves10

Reply:
Suppose that a draft message loses its focus while keyboard is hidden. There is no other drafts. In this case, the composer should be shown because there is no focused drafts. But keyboardDidHide event isn't emitted and composer isn't shown